cabal.project

```diff
@@ -191,12 +191,13 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-ledger
-  tag: 1db68a3ec0a2dcb5751004beb22b906162474f23
-  --sha256: 03pv2jvskbi65dwaddp6a8bxbbcw674csjxhg8xbqd6q1kfpc41a
+  tag: 461506591729379168a607c3298746d0784621f1
```

Comment:
Could we make sure this fix is included? https://github.com/input-output-hk/cardano-ledger/pull/2771

Reply:
I'm also assuming we are waiting on the time bug fix as well?

Reply:
Let's add the vrf check fix to the list.